m4/fontforge_arg_enable.m4

```diff
@@ -214,7 +219,7 @@ if test x$use_gdk = xyes ; then
             AC_MSG_NOTICE([building the GUI with the GDK2 backend...])
         ],
         [
-            AC_MSG_ERROR([Cannot build GDK backend without GDK installed.])
+            AC_MSG_ERROR([Cannot build GDK backend without GDK installed. Please install the GDK Developer Package.])
```

Comment:
There's no such thing, you'd need to install GTK development libraries to get GDK.

Reply:
If you want to go ahead and rewrite a correct version, go ahead, and close both pull requests.
For now, I'm still running command line right now (lacking GUI stuff) and haven't followed up on loading a full GUI yet, ...but at the rate I'm going now, it may be next year before I get there.  :-/